redcycle/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from uuid import uuid4
+from .enums.CategoryEnum import CategoryEnum
+
+class Account(models.Model):
+    uuid = models.CharField(primary_key=True, max_length=36)
+    firstName = models.CharField(max_length=100)
+    lastName = models.CharField(max_length=100)
+    email = models.CharField(max_length=100)
+    active = models.BooleanField(default=True)
+    def __str__(self):
+        return self.uuid
```

Comment:
Here is where the models are being defined. Each class attribute has a field (CharField, BooleanField, etc) that allows Django to map the attribute to a database field with the necessary constraints.